src/containers/sidebar/index.css

```diff
@@ -0,0 +1,62 @@
+.rectangle-8 {

```

Comment:
1. Use descriptive names for your css classes. Currently it is not clear how rectangle-8 differs from rectangle-5 etc.
2. All these classes are defined in this css file, but are not used anywhere in the sidebar component? Why do you have these classes?

Reply:
Alright cleaning those up.